modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -1987,7 +1987,7 @@ transform.
 
 @param image 8-bit, single-channel binary source image. The image may be modified by the function.
 @param lines Output vector of lines. Each line is represented by a 2 or 3 element vector
-\f$(\rho, \theta)\f$ or \f$(\rho, \theta, \votes)\f$ . \f$\rho\f$ is the distance from the coordinate origin \f$(0,0)\f$ (top-left corner of
+\f$(rho, theta)\f$ or \f$(rho, theta, votes)\f$ . \f$\rho\f$ is the distance from the coordinate origin \f$(0,0)\f$ (top-left corner of
```

Comment:
`\rho` and `\theta` can be rendered correctly in the [doc](https://docs.opencv.org/3.4.3/dd/d1a/group__imgproc__feature.html#ga46b4e588934f6c8dfd509cc6e0e4545a).
Only `\votes` should be changed to `votes` in my opinion.